editor/symbol_tooltip.h

```diff
@@ -0,0 +1,84 @@
+/*  symbol_tooltip.h                                                      */
+/**************************************************************************/
+/*                         This file is part of:                          */
+/*                             GODOT ENGINE                               */
+/*                        https://godotengine.org                         */
+/**************************************************************************/
+/* Copyright (c) 2014-present Godot Engine contributors (see AUTHORS.md). */
+/* Copyright (c) 2007-2014 Juan Linietsky, Ariel Manzur.                  */
+/*                                                                        */
+/* Permission is hereby granted, free of charge, to any person obtaining  */
+/* a copy of this software and associated documentation files (the        */
+/* "Software"), to deal in the Software without restriction, including    */
+/* without limitation the rights to use, copy, modify, merge, publish,    */
+/* distribute, sublicense, and/or sell copies of the Software, and to     */
+/* permit persons to whom the Software is furnished to do so, subject to  */
+/* the following conditions:                                              */
+/*                                                                        */
+/* The above copyright notice and this permission notice shall be         */
+/* included in all copies or substantial portions of the Software.        */
+/*                                                                        */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,        */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF     */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. */
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY   */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,   */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE      */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
+/**************************************************************************/
+
+#ifndef SYMBOL_TOOLTIP_H
+#define SYMBOL_TOOLTIP_H
+
+#include "code_editor.h"
+#include "modules/gdscript/gdscript_parser.h"
+#include "modules/gdscript/language_server/gdscript_extend_parser.h"
+#include "scene/gui/box_container.h"
+#include "scene/gui/panel_container.h"
+#include "scene/gui/rich_text_label.h"
+
+class SymbolTooltip : public PanelContainer {
+	GDCLASS(SymbolTooltip, PanelContainer);
+
+	CodeTextEditor *code_editor = nullptr;
+	CodeEdit *text_editor = nullptr;
+	PanelContainer *panel_container = nullptr;
+	VBoxContainer *layout_container = nullptr;
+	TextEdit *header_label = nullptr;
+	RichTextLabel *body_label = nullptr;
+	Timer *tooltip_delay = nullptr;
+	String last_symbol_word;
+	bool mouse_inside;
+
+	String _get_doc_of_word(const String &symbol_word);
```

Comment:
```suggestion
	String _get_doc_of_word(const String &p_symbol_word);
```
Same in the rest of this and the CPP file, always prefix arguments with `p_`, unless they are return arguments, then use `r_`

Reply:
I've updated all parameters to be prefixed with 'p_'.